### PR TITLE
Test replicates

### LINKED ISSUE #477: create a test for samplesheet with technical replicates
test to create CAT_FASTQ module

### PR DESCRIPTION
**Pull request by sateeshperi**
closes #477

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,8 +16,9 @@
 - 🔧 reorg individual configs to `conf/modules/` named configs [#459](https://github.com/nf-core/methylseq/pull/469)
 - 🔧 `run_preseq` param + skip preseq/lcextrap module by default [#458](https://github.com/nf-core/methylseq/pull/470)
 - 🔧 `run_qualimap` param + skip qualimap module by default [#367](https://github.com/nf-core/methylseq/pull/471)
-- 🔧 Raised Nextflow version requirement to `24.10.2`
+- 🔧 Raised Nextflow version requirement to `24.10.2` in CI
```
Comment (GallVp) [inline CHANGELOG.md:19]: This was actually also applied to `manifest` which means that users can only use the pipeline with Nextflow >= 24.10.2

Comment (sateeshperi) [reply CHANGELOG.md:19]: I see...cool....I ll update this in the next release PR

File: workflows/methylseq/main.nf [hunk 2]
```diff
@@ -199,6 +213,10 @@ workflow METHYLSEQ {
     )
 
     emit:
+    bam            = ch_bam                      // channel: [ path(bam) ]
```
Comment (GallVp) [inline workflows/methylseq/main.nf:216]: // channel:  [ val(meta), path(bam) ]

and same applies to others, perhaps?

**Review by GallVp — APPROVED** [review]

File: workflows/methylseq/main.nf (hunk 2 above)
Comment (sateeshperi) [reply workflows/methylseq/main.nf:216]: will update in next. Thanks

**Pull request merged**